.github/workflows/build.yml

```
@@ -0,0 +1,23 @@
+name: Build
+
+on: 
+  pull_request:
+      branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest 
```

should we pin it? Latest is changing https://github.com/actions/virtual-environments/issues/1816

```suggestion
    runs-on: ubuntu-20.04
```